homeassistant/components/nissan_leaf/switch.py

```diff
@@ -92,6 +95,13 @@ def is_on(self):
         if await self.car.async_start_charging():
             self.car.data[DATA_CHARGING] = True
 
+    # @MartinHjelmare says should be removed if we don't support it.
+    #                 Maybe it should be a scene?
+    # Unsure if better to provide as a sensor for the state, and a service to
+    # start a charge, e.g.
+    # - service: nissan_leaf.start_charging
```

Comment:
Yes! Service + sensor is a good idea! :+1:

The service should be registered in the nissan_leaf component.

Reply:
Converted to a service and sensor in commit https://github.com/home-assistant/home-assistant/pull/21145/commits/9fa7369984593be83b65fc1c1115d1919103e64c